tests/integration/test_native_iq_integration.py

```
@@ -0,0 +1,339 @@
+    file_id = getattr(file_obj, "id", None)
+    if not file_id:
+        return None
+
+    # Download to temp file
+    tmp_dir = Path(tempfile.gettempdir()) / "ni_integration_attachments"
+    tmp_dir.mkdir(parents=True, exist_ok=True)
+    out_path = tmp_dir / f"{name}"
+
+    # Download using internal service method
+    try:
+        ok = drive_service.download_file(file_id, str(out_path))
```

Ruff (0.12.2)

Local variable `ok` is assigned to but never used

Remove assignment to unused variable `ok`

(F841)